site/layouts/shortcodes/scss-docs.html

```diff
@@ -39,5 +41,16 @@
     {{- $match = replace $match " !default" "" -}}
   {{- end -}}
 
-  {{- highlight $match "scss" "" -}}
+  {{- $output := "" -}}
+  {{- if (gt $unindent 0) -}}
+    {{- $prefix := (strings.Repeat $unindent " ") -}}
+    {{- range $line := split $match "\n" -}}
+      {{- $line = strings.TrimPrefix $prefix $line -}}
+      {{ $output = printf "%s%s\n" $output $line }}
+    {{- end -}}
+    {{- $output = chomp $output -}}
+  {{- else -}}
+    {{- $output = $match -}}
+  {{- end -}}
+  {{- highlight $output "scss" "" -}}
```

Comment:
Just found this, it might help having an automatic way to handle indentation. I don't know if we can do better with Hugo.

```html
  {{- $unindent := 0 -}}
  {{- $found := false -}}
  {{- $first_line:= index (split $match "\n") 0 -}}
  {{- range $char := split $first_line "" -}}
    {{- if and (eq $char " ") (not $found) -}}
      {{- $unindent = add $unindent 1 -}}
    {{- else -}}
      {{- $found = true -}}
    {{- end -}}
  {{- end -}}
  {{- $output := "" -}}
  {{- if (gt $unindent 0) -}}
    {{- $prefix := (strings.Repeat $unindent " ") -}}
    {{- range $line := split $match "\n" -}}
      {{- $line = strings.TrimPrefix $prefix $line -}}
      {{ $output = printf "%s%s\n" $output $line }}
    {{- end -}}
    {{- $output = chomp $output -}}
  {{- else -}}
    {{- $output = $match -}}
  {{- end -}}
  {{- highlight $output "scss" "" -}}
```

This way, there no impact on the shortcode calls, no new parameter.

Reply:
Probably a new PR

Reply:
Done via #38308